en/faq.html

```diff
@@ -317,7 +331,31 @@ <h1>
                             <a href="faq.html#Can-I-present-my-childs-or-parents-SMART-Health-Card-on-their-behalf" class="question">Can I present my child's/parent's SMART Health Card on their behalf? </a>
                         </p>
                         <p class="answer">
-                            Yes, as long as you are allowed to. Sharing a SMART Health Card is similar to sharing other medical records in this way.
+                            Yes, as long as you have permission to do so. Sharing a SMART Health Card is similar to sharing other medical records in this way.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="row mt-3">
+                    <div class="col-12">
+                        <a name="Are-SMART-Health-Cards-only-for-Covid-19-vaccination-verification" class="anchor"></a>
+                        <p class="question">
+                            <a href="faq.html#Are-SMART-Health-Cards-only-for-Covid-19-vaccination-verification" class="question">Are SMART Health Cards only for COVID-19 vaccination verification? </a>
+                        </p>
+                        <p class="answer">
+                            No. The SMART Health Card Framework was designed to empower individuals with access to their healthcare data in secure, easy-to-access, equitable, and privacy-preserving ways. Developed prior to the COVID-19 pandemic by researchers at public and private institutions, the creators are honored that the technology has helped organizations and people worldwide at an urgent and critical time. The power of the SMART Health Cards Framework is its open-sourced standard that can be applied to many use cases beyond COVID-19. Research and planning is currently underway for uses like childhood immunization records. Visit this Harvard Medical School article for more information about the SMART Health Cards Framework.
```

Comment:
This answer needs review. There are some factual and technical issues here. Do we want to edit inline in this pull request or out of band?